### DISCUSSION TITLE
Is public channel like discord, github or gitter available for question/discussion?

### DISCUSSION THREAD

**Discussion by hellosagar** [robolectric/robolectric#7105] [General]
Or in Robolectric using GitHub discussion section for that purpose?

**Comment by utzcoz** [comment]
Yup. GitHub Discussion is used for generic discussion, e.g., ideas, announcement and potential problems.  If discussion indicates a problem or feature of Robolectric, we can create a related issue for it. And then, we can move tech detail about problem fixing/feature implementation to issue/PR page, and update final result to discussion lastly.

There is another channel for discussion: groups.google.com/group/robolectric. You can post similar things at Robolectric's Google Groups. But I prefer to use GitHub Discussion to precipitate knowledge from discussion, e.g. FQA on GitHub, along with other existed knowledge, e.g. documentation and example.

I also want GitHub Discussion becomes a place to share practice and successfully story to use Robolectric. For good practice, we can discuss whether to re-post it on http://robolectric.org/.

Those are my personal thoughts about GitHub Discussion, and the reason I encourage to enable GitHub Discussion. I think @hoisie can help to provide more detailed future plan about public channel for Robolectric's discussion.

**Reply by utzcoz** [reply]
There is an example to use GitHub search to search contents of GitHub Discussion: https://github.com/robolectric/robolectric/search?q=GSoC&type=discussions.
